boards/arm/mimxrt1170_evk/board.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2021, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+board_runner_args(pyocd "--target=mimxrt1170_cm7")
+board_runner_args(jlink "--device=MIMXRT1176xxxA_M7")
```

Comment:
Will this work for the M4 too?

Reply:
I have modified as lpcxpresso55s69 doing.